.github/workflows/release.yaml

```diff
@@ -11,11 +11,11 @@ jobs:
     if: github.repository == 'aws/karpenter'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
         with:
           fetch-depth: 0
       - name: Create GitHub Release
-        uses: 'marvinpinto/action-automatic-releases@latest'
+        uses: marvinpinto/action-automatic-releases@919008cf3f741b179569b7a6fb4d8860689ab7f0 # latest
```

Comment:
Let's use a pinned version rather than something that's on latest that will change over time

Reply:
I was hesitant to downgrade since the last tagged release (v1.2.1) was from over two years ago and there have been several fixes since. I suppose we'll see in the karpenter-core release if there are any issues with v1.2.1 though and it should be easy enough to revert if there are.